JackyZhang1219/books-and-breakfast-app
# added tracker link and fixed alert when no school is selected

### Description

I added the tracker link when you press the tracker button and also added an alert for when you haven't selected a school yet.

### QA

Please explain in detailed steps how the reviewer can test your changes and what the expected behaivors are (visual would help!). Consider different test cases/scenarios.

Should open tracker link on google docs when the tracker button is pressed (before, it was routed to an empty Tracker screen). When a school isn't selected from the dropdown, an alert should pop up and it shouldn't navigate to the school until the school is selected.

### Packages

Please describe any additional packages you added to package.json.

### Checklist
